src/libraries/System.Net.WebHeaderCollection/ref/System.Net.WebHeaderCollection.cs

```diff
@@ -97,7 +97,9 @@ public partial class WebHeaderCollection : System.Collections.Specialized.NameVa
         public void Add(System.Net.HttpRequestHeader header, string? value) { }
         public void Add(System.Net.HttpResponseHeader header, string? value) { }
         public void Add(string header) { }
```

Comment:
All these presumably complicate things next time we run the tool to update the refs.